blockbase/functions.php

```diff
@@ -14,6 +14,9 @@ function blockbase_support() {
 		// Add support for editor styles.
 		add_theme_support( 'editor-styles' );
 
+		// Enqueue the Gutenberg plugin theme.css file
+		add_theme_support( 'wp-block-styles' );
```

Comment:
Is the addition of `wp-block-styles` necessary to achieve this change?  It adds a lot of CSS to the theme and I'm hesitant to do that late in the game.  I know we evaluated it elsewhere and I believe we decided not to bring that into Blockbase.

Reply:
Yep, it is necessary. But I reverted the Gutenberg styles inclusion in Blockbase to explore that change in a separated PR because, as you said, that change can have a broader impact.